pydrive2/fs/spec.py

```diff
@@ -49,13 +55,120 @@ def should_retry(exc):
     )(func)
 
 
+def _cache_file(client_id):
+    cache_dir = appdirs.user_cache_dir("pydrive2fs", "iterative")
```

Comment:
let's not make it about iterative, wdyt? (it'll be part of the file name / path at least on some system, right?)

Reply:
@shcheklein good point, forgot about this when porting. I guess we could just refuse to set org. Let me check...